### PR TITLE
Update Flink Roadmap

### PR DESCRIPTION
**Pull request by wuchong**
This updates the roadmap of the website, as discussed in this thread: 
https://lists.apache.org/thread/szdr4ngrfcmo7zko4917393zbqhgw0v5

Note: The Chinese language version of the page needs to be translated / adjusted as well.

### CODE REVIEW

**Comment by xiangyuf** [comment]
@wuchong Hi Jark, shall we add 'support short-lived jobs' to the roadmap radar svg?

**Comment by wuchong** [comment]
@xiangyuf I think it's hard to say it's a feature, but a use case. The short-lived jobs are batch jobs and is already covered by `DataStream(batch)`.

**Comment by xiangyuf** [comment]
> @xiangyuf I think it's hard to say it's a feature, but a use case. The short-lived jobs are batch jobs and is already covered by `DataStream(batch)`.

Thanks for reply. LGTM now.

File: docs/content/roadmap.md [hunk 1]
```diff
@@ -25,19 +25,14 @@ under the License.
...
 
 More details and various smaller changes can be found in the
 [FLIPs](https://cwiki.apache.org/confluence/display/FLINK/Flink+Improvement+Proposals)
 
-The roadmap is continuously updated. New features and efforts should be added to the roadmap once
-there is consensus that they will happen and what they will roughly look like for the user.
+The roadmap is continuously updated. New features and efforts should be added to the roadmap once there is consensus that they will happen and what they will roughly look like for the user.
 
-**Last Update:** 2022-11-14
+**Last Update:** 2023-08-XX
```
Comment (xintongsong) [inline docs/content/roadmap.md]: Just a reminder for updating the date on merging.

**Review by xintongsong — APPROVED** [review]
Thanks @wuchong for driving this effort. The contents LGTM. I've also tried build the site locally and everything looks fine.

**Comment by wuchong** [comment]
Thank you @xintongsong , I will merge it.

**Pull request merged**